modules/xfem/test/include/postprocessors/ParisLaw.h

```diff
@@ -0,0 +1,47 @@
+//* This file is part of the MOOSE framework
```

Comment:
We should think a little bit about what type of object this should be. Generally, a postprocessor just computes one value. While this does compute one value, it also does more than that. Maybe a GeneralUserObject would be better here.